components/ArrowNavLink/ArrowNavLink.jsx

```diff
@@ -33,14 +32,6 @@ export function ArrowNavLink({
         if (Number.isNaN(nextCatalogIndex)) {
             return nextCatalogIndex - 1
         }
-        if (nextCatalogIndex >= children.length) {
-            title = 'Назад к руководствам'
-            href = { pathname: '/' }
-        } else {
-            const nextCatalog = getCatalogOptions(children[nextCatalogIndex])
-            title = nextCatalog.title
-            href.query.pageUrl = [nextCatalog.url]
-        }
```

Comment:
This hasn't been used for since the primary page of the manuals is the first page